_sass/variables.scss

```diff
@@ -2,6 +2,9 @@
 $large-breakpoint: 64em !default;
 $medium-breakpoint: 42em !default;
 
+//Navbar color (cell-phones)
+$theme_color: #157878 !default;
```

Comment:
The problem is that this is a Sass variable, and the `site.theme_color` above is a Jekyll variable. The above will need to be set via the site's `_config.yml`. We could also set that as a Sass variable if we wanted to make it available to Sass, but I don't believe it's used.